src/components/variable.jsx

```diff
@@ -107,7 +109,7 @@ const Variable = React.createClass({
                 {startDate && stopDate &&
                   <h3>Du <FormattedDate value={startDate} /> au <FormattedDate value={stopDate} />&nbsp;:</h3>
                 }
-                <pre><code>{formulas[date].content}</code></pre>
+                <code><SyntaxHighlighter language='python' style={tomorrow}>{formulas[date].content}</SyntaxHighlighter></code>
```

Comment:
Something that looks like HTML. For now, it will include `Link`. 
Example:  
```
<code><SyntaxHighlighter language='python' style={tomorrow}><Link key="toto" to="#">HOME</Link></SyntaxHighlighter></code>
```

The thing is that if we send this Link in a string, it will not be rendered as a clickable component.